biotrainer/utilities/fasta.py

```diff
@@ -127,3 +127,54 @@ def read_FASTA(path: str) -> List[SeqRecord]:
         raise  # Already says "No such file or directory"
     except Exception as e:
         raise ValueError(f"Could not parse '{path}'. Are you sure this is a valid fasta file?") from e
+
+
+def hf_to_fasta(
+    sequences: List[str],
+    targets: List[Any],
+    set_values: List[str],
+    sequences_file_name: str,
+    targets_file_name: Optional[str] = None
```

Comment:
See above, I'd change this to provide the huggingface dataset and all information necessary to convert it to fasta files.